api/src/index.ts

```diff
@@ -1,22 +1,25 @@
-import { writeFileSync } from "fs";
 import Fastify from "fastify";
 import fetch from "node-fetch";
 import { generateNbTxsPerDay } from "./tasks/generateNbTxsPerDay";
 import { generateUniqueAddressGrowingPerDay } from "./tasks/generateUniqueAddressGrowingPerDay";
+import { readData, writeData } from "./utils";
 
 let cacheData: any = false;
 async function generateDataStats() {
+  const currentData = readData();
```

Comment:
currentData will be undefined first time server is run, so line 11 would crash the server